ddgr

```diff
@@ -1454,6 +1460,14 @@ class DdgCmd(object):
                 elif cmd == 'x':
                     Result.urlexpand = not Result.urlexpand
                     printerr('url expansion toggled.')
+                elif cmd[0] == 'c' and cmd[2:].isdigit():
+                    url = self._urltable[cmd[2:]]
+                    try:
+                        pyperclip.copy(url)
+                    except NameError:
+                        printerr("cannot copy url: pyperclip not installed")
```

Comment:
Please change to:

`'cannot copy url: pyperclip missing'`

Note the single quotes.